src/content/pages/volunteers.mdx

```diff
@@ -95,13 +95,13 @@ There are always some tasks which we simply cannot plan beforehand. Based on wha
 
 # How to sign up
 
-Please acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6) first, and then fill out the form below to register your interest in volunteering at EuroPython 2024!
-We will get back to you with more details and information on how to proceed.
-Thanks ❤️
+Quite a lot of enthusiastic Pythonistas expressed their interest to volunteer and with so many great applications,
+making a decision was tough this year. Unfortunately, at this point we are not able to offer you a ticket for your volunteer contribution.
+We have a limited number of tickets for volunteers and we are unable to accommodate everyone.
+
+If you can make it to the conference and you’d like to volunteer some of your time we’d be truly grateful. Even though we cannot offer you a ticket,
+we will gladly welcome you to our volunteer family! ❤️ If you are interest, you can mail us at [volunteers@europython.eu](mailto:volunteers@europython.eu)
```

Comment:
modified it a bit to hopefully make it a bit natural and explicit. 
```suggestion
Thank you to all the enthusiastic Pythonistas who expressed interest in volunteering! With so many great applications, making a decision was tough this year. Unfortunately, we have already allocated all our volunteer tickets and are no longer able to offer you one at this time. Due to our limited capacity, we cannot accommodate everyone.

However, if you're able to attend the conference and still wish to volunteer, we would truly appreciate your support. While we cannot provide a ticket, we warmly welcome you to join our volunteer family! ❤️ If you're interested, please email us at [volunteers@europython.eu](mailto:volunteers@europython.eu).
```

Reply:
Thank you!